### PR TITLE
Adding instructor note about piloting to learn

### LINKED ISSUE #96: add instructor note framing teaching as a learning process
A recurring theme in the first pilot was discussion of the benefits of piloting a new lesson and treating teaching as an opportunity to learn and improve. It would be good to add a general Instructor Note (i.e. to `instructors/instructor-notes.md`) encouraging Trainers to frame the discussion this way and to draw on their own experience when talking about the ways in which piloting a lesson can help you improve it.

### PR DESCRIPTION
**Pull request by sstevens2**
Drafted a bit of text in the instructor notes on the reoccurring theme of piloting as a learning opportunity.
It is a very quick draft based on the issue so feedback highly welcome.  I was especially undecided if I should write it as a direct recommendation to the reader using "you" or not.

Closes #96

### CODE REVIEW

File: instructors/instructor-notes.md [hunk 1]
```diff
@@ -2,4 +2,16 @@
...
 
-This is a placeholder file. Please add content here. 
+### Training Themes
+
+#### Lesson Development as a Learning Process
+
+A recurring theme in this training is the discussion of the benefits of piloting a new 
+lesson and treating teaching as an opportunity to learn and improve the lesson. 
+You should try to frame the discussion this way and to draw on your own experience 
+when talking about the ways in which piloting a lesson can help you improve it.
```
Comment (tobyhodges) [inline instructors/instructor-notes.md:12]:
```suggestion
We have found it helpful to frame the discussion this way and to draw on our own experience 
when talking about the ways in which piloting a lesson can help to improve it.
```

File: instructors/instructor-notes.md [hunk 2]
```diff
@@ -2,4 +2,16 @@
...
+For example, when teaching the assessments or narrative section you might discuss a time 
+when you piloted a workshop and realize there was a concept missing in the demo/narrative that the learners needed in order to be able to complete the exercise.
```
Comment (tobyhodges) [inline instructors/instructor-notes.md:14]:
```suggestion
when you piloted a workshop and realized there was a concept missing in the demo/narrative
that learners needed in order to be able to complete the exercise.
```

**Review by tobyhodges — CHANGES_REQUESTED** [review]
I like this note, and I think some "you could"/"you might" is good. But I have reworded to remove the "you should" because I expect other Trainers will want to frame things differently and I don't want to discourage that.

File: instructors/instructor-notes.md (hunk 1 above)
Comment (sstevens2) [reply instructors/instructor-notes.md:12]: Good idea!

**Review by tobyhodges — APPROVED** [review]

**Comment by tobyhodges** [comment]
Thanks @sstevens2 👍

**Pull request merged**